### ISSUE TITLE
Missing Site Search

### ISSUE THREAD

**Issue by anilnatha** [MAAP-Project/ZenHub#397]
The [21st Century Integrated Digital Experience Act](https://digital.gov/resources/21st-century-integrated-digital-experience-act/) contain requirements that public facing government websites should meet.

One of those being providing a search function for users to easily find content.  At one point we did have a search function as shown in an older instance at [www.maap.xyz](https://www.maap.xyz); however that feature doesn't appear to be present in the latest iterations on [Pilot OPS](https://www.maap-project.org) or [GCC OPS](https://ops.maap-project.org).

**Comment by anilnatha** [comment]
I've determined how to enable the search functionality in the portal. Turns out it's a feature of the Monochrome Pro theme that we are using. And while doing this research I also learned that the search feature isn't being rendered correctly.  The search feature should occupy the same region as the header, but it's breaking out of that and occupying the entire page. 

So I'm now trying to figure out how to fix this issue.

**Comment by anilnatha** [comment]
I've found the CSS that was causing the issue:

```css
.site-header {
    position: static;
    box-shadow: inherit;
}
```

I believe this style was added to remove the fixed header feature provided by the Monochrome Pro theme.  Until I can come up with a better solution, I've commented the above CSS so that I can enable the search functionality without it breaking the page layout.

The search feature is working on all three environments.
